packages/rum-core/src/domain/action/actionCollection.ts

```diff
@@ -62,7 +62,7 @@ function processAction(
     ? {
         action: {
           id: action.id,
-          loading_time: toServerDuration(action.duration),
+          loading_time: discardNegativeLoadingTime(toServerDuration(action.duration)),
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏You can probably reuse `discardNegativeDuration` from `viewCollection` https://github.com/DataDog/browser-sdk/blob/146b4bf68e61b8ce24e563b85bc930432eacf69f/packages/rum-core/src/domain/view/viewCollection.ts#L135